scripts/inactive_issue_checker.py

```diff
@@ -0,0 +1,492 @@
+
+            for pr in pull_requests['nodes']:
+                pr_number = pr['number']
+                linked_issues = pr['closingIssuesReferences']['nodes']
+                if linked_issues:
+                    for issue in linked_issues:
+                        issue_number = issue['number']
+                        if issue_number not in issue_to_prs:
+                            issue_to_prs[issue_number] = set()
+                        issue_to_prs[issue_number].add(pr_number)
+
+            page_info = pull_requests['pageInfo']
+            has_next_page = page_info['hasNextPage']
+            req_credentials['cursor'] = page_info['endCursor']
+
+        return issue_to_prs
+
+    def unassign_issue(self, issue: Issue) -> bool:
+        """Unassigns a user from an issue.
+
+        Args:
+            issue: Issue. The issue to unassign.
+
+        Returns:
+            bool. Whether the unassignment was successful.
+
+        Raises:
+            AssertionError. Raised if the response from the request is None.
+            requests.HTTPError. Raised if the request fails.
+        """
+        if not issue.assignee_username:
+            return False
+
+        url = f'{self.base_url}/issues/{issue.number}/assignees'
+        response = requests.delete(
+            url,
+            headers=self.rest_headers,
+            json={'assignees': [issue.assignee_username]},
+            timeout=10
+        )
+        if response is None:
+            raise AssertionError('Received null res while unassigning issue')
+
+        response.raise_for_status()
+
+        return response.status_code == 200
+
+    def add_alert_comment_on_issue(self, issue: Issue) -> None:
+        """Posts unassignment comment on an issue.
+
+        Args:
+            issue: Issue. The issue to comment on.
+
+        Raises:
+            AssertionError. Raised if the response from the request is None.
+            requests.HTTPError. Raised if the request fails.
+        """
+        url = f'{self.base_url}/issues/{issue.number}/comments'
+        comment = (
+            f'Hi @{issue.assignee_username} this PR is inactive '
+            f'for {INACTIVE_DAYS_THRESHOLD} and you will be '
+            f'unassigned soon if no activity is done.\n '
+            f'If you are still working on this PR, '
+            f'please make a follow-up commit within'
+            f'{UNASSIGN_DAYS_THRESHOLD-INACTIVE_DAYS_THRESHOLD} '
+            f'(and submit it for review, if applicable). '
+            f'Please also let us know if you are stuck so we can help you!'
+        )
+        response = requests.post(
+            url,
+            headers=self.rest_headers,
+            json={'body': comment},
+            timeout=10
+        )
+        if response is None:
+            raise AssertionError('Received null res while commenting on issue')
+        response.raise_for_status()
+
+    def remove_assignee_comment_on_issue(self, issue: Issue) -> None:
```

Comment:
This name suggests it's removing something but you are actually adding a comment. Please fix.